Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/3scale/system-builder:ruby25-oracle
+FROM quay.io/3scale/system-builder:ruby25-bundler-2.2.18
```

Comment:
we don't need something oracle specific here @hallelujah ?

Reply:
Good spotting it! But we do not need because the branch was merged into ruby-25 already.